### PR TITLE
PYMODM-96 Url Schemes added in docs

### PR DESCRIPTION
**Pull request by shreybatra**
URL schemes which are accepted ['http', 'https', 'ftp', 'ftps'] added to documentation.

### CODE REVIEW

**Comment by prashantmital** [comment]
LGTM.

**Comment by shreybatra** [comment]
@prashantmital @ShaneHarvey ??

File: pymodm/fields.py [hunk 1]
```diff
@@ -516,7 +516,7 @@ def validate_ip_address(value):
 
 
 class URLField(MongoBaseField):
-    """A field that stores URLs."""
+    """A field that stores URLs. It accepts the following Schemes only - ['http', 'https', 'ftp', 'ftps']"""
```
Comment (ShaneHarvey) [inline pymodm/fields.py]: Generally we keep lines to 80 characters. Can you change this to:
```
"""A field that stores URLs.

This field only accepts http, https, ftp, and ftps URL schemes.
"""
```

**Comment by shreybatra** [comment]
@ShaneHarvey Done

**Comment by shreybatra** [comment]
@ShaneHarvey @prashantmital Please review.

**Comment by prashantmital** [comment]
Thanks for the pull @shreybatra. Apologies for the delay in merging this.

**Pull request merged**